Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	phyluce_run1
	2

[Wed Mar 23 15:11:53 2022]
rule phyluce_run1:
    input: /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.fasta, /home/arthur/Bureau/postdocs/isem/hybscan/pipeline/tools/uce-probes.fasta
    output: /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce, /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce.fasta
    jobid: 1
    wildcards: PATH=/home/arthur/Bureau/postdocs/isem/hybscan/data, ID=SRR5437981, EXT=.fastp.megahit.

[Wed Mar 23 15:11:53 2022]
Error in rule phyluce_run1:
    jobid: 1
    output: /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce, /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce.fasta

RuleException:
CalledProcessError in line 134 of /home/arthur/Bureau/postdocs/isem/hybscan/pipeline/Snakefile:
Command ' set -euo pipefail;  set +eu;. $(conda info --base)/etc/profile.d/conda.sh;conda activate phyluce_env;mkdir /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/input; sed 's/[Y|S|W|K|M|R|y|s|w|k|m|r]/N/g' /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.fasta | awk '/>/{sub(">contig", ">NODE_", $0); sub("$", "_length_100_cov_10", $0); print $0}; !/>/{print}' > /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/input/SRR5437981.fastp.megahit.contigs.fasta; phyluce_assembly_match_contigs_to_probes --contigs /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/input --probes /home/arthur/Bureau/postdocs/isem/hybscan/pipeline/tools/uce-probes.fasta --output /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/output;echo '[all]
SRR5437981.fastp.megahit.' > /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/output/SRR5437981.fastp.megahit..conf';phyluce_assembly_get_match_counts --locus-db /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/output/probe.matches.sqlite --taxon-list-config /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/output/SRR5437981.fastp.megahit..conf --taxon-group 'all' --incomplete-matrix --output /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce/output/SRR5437981.fastp.megahit..match.conf;touch /home/arthur/Bureau/postdocs/isem/hybscan/data/SRR5437981.fastp.megahit.phyluce.fasta; ' returned non-zero exit status 1.
  File "/home/arthur/Bureau/postdocs/isem/hybscan/pipeline/Snakefile", line 134, in __rule_phyluce_run1
  File "/home/arthur/bin/miniconda/envs/hybscan_env/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/arthur/Bureau/postdocs/isem/hybscan/pipeline/.snakemake/log/2022-03-23T151153.495838.snakemake.log
